helm-charts/chatqna/gaudi-values.yaml

```diff
@@ -30,11 +40,14 @@ llm-uservice:
     resources:
       limits:
         habana.ai/gaudi: 1
+
 global:
   http_proxy:
   https_proxy:
   no_proxy:
   HUGGINGFACEHUB_API_TOKEN: "insert-your-huggingface-token-here"
-  volume: /mnt
   LANGCHAIN_TRACING_V2: false
   LANGCHAIN_API_KEY: "insert-your-langchain-key-here"
+  # set modelUseHostPath to host directory if you want to use hostPath volume for model storage
+  # comment out modeluseHostPath if you want to download the model from huggingface
+  modelUseHostPath: /mnt
```

Comment:
`/mnt` is general mount point where other unrelated things can/will be mounted.  Better to specify a subdirectory (e.g. `/mnt/opea-models`).

Reply:
User should override this directory with the one contains cached models at deployment time.
The /mnt is used just to avoid any reporting errors like "directory not exist".
For a production ready deployment, PV/PVC should be used.

Reply:
If user is expected to override it, getting an error from the default **is** a good thing!

Reply:
> For a production ready deployment, PV/PVC should be used.

Agree, but quick `git grep PVC` for the OPEA projects did not find _any_ mentions of that.

Reply:
PV/PVC will be tracked by another issue #128